modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -5,20 +5,72 @@ This module is included in the following assemblies:
+You can expose tracing data to the {TempoName} by appending a named element and the `opentelemetry` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMeshControlPlane`. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the OpenTelemetry Collector endpoint.
 
-You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+//As of July 2, 2024, there is no common attribute for OpenTelemetry Collector.
+
+You can create a {OTELName} instance in a mesh namespace and configure it to send tracing data to a tracing platform backend service.
 
 .Prerequisites
 
-* You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace.
-* You have created namespaces such as `istio-system` and `tracing-system`.
+* You have created a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing {TempoName}" in the "Additional resources" section.
+
+* You have installed the {OTELOperator} in either the recommended namespace or the namespace `openshift-operators`. For more information, see "Installing the {OTELName}" in the "Additional resources" section.
+
+* If using Service Mesh 2.5 or earlier, set `spec.tracing.type` to `None` so tracing data can be sent to the OpenTelemetry Collector.
 
 .Procedure
 
-. Configure the `ServiceMeshControlPlane` resource to define an extension provider:
+. Create an OpenTelemetry Collector instance in a mesh namespace. This example uses the `bookinfo` namespace:
++
+[source, yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: bookinfo  # <1>
+  annotations:
+    sidecar.istio.io/inject: 'true' # <2>
+spec:
+  mode: deployment
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+    exporters:
+      otlp:
+        endpoint: "tempo-sample-distributor.tracing-system.svc.cluster.local:4317" # <3>
+        tls:
+          insecure: true
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: []
+          exporters: [otlp]
+----
+<1> Include the namespace in the `ServiceMeshMemberRoll` member list.
+<2> The sidecar injection annotation is only required when you have enabled the `spec.security.dataPlane` parameter for mTLS encryption in the `ServiceMeshControlPlane` resource.
+<3> In this example, a TempoStack instance is running in the `tracing-system` namespace. You don't have to include the TempoStack namespace e.g. `tracing-system` in the `ServiceMeshMemberRoll` member list.
```

Comment:
```suggestion
<3> In this example, a TempoStack instance is running in the `tracing-system` namespace. You do not have to include the TempoStack namespace, such as`tracing-system`, in the `ServiceMeshMemberRoll` member list.
```
OK, contractions are allowed, https://www.ibm.com/docs/en/ibm-style?topic=grammar-contractions, but I personally don't like them in documentation. Up to you.

"e.g." takes commas like a clause; ,e.g.,
But I have no idea if either e.g. or i.e translate to, say, Japanese. So better to spell out either intention.
https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference